dev/tools/gen_keycodes/bin/gen_keycodes.dart

```diff
@@ -17,7 +17,7 @@ import 'package:gen_keycodes/utils.dart';
 /// Get contents of the file that contains the key code mapping in Chromium
 /// source.
 Future<String> getChromiumConversions() async {
-  final Uri keyCodeMapUri = Uri.parse('https://cs.chromium.org/codesearch/f/chromium/src/ui/events/keycodes/dom/keycode_converter_data.inc');
+  final Uri keyCodeMapUri = Uri.parse('https://cs.chromium.org/codesearch/f/chromium/src/ui/events/keycodes/dom/dom_code_data.inc?cl=c3b5578091537d177f21b63732c89db3dd7b70e3');
```

Comment:
Thanks for updating this, but why tie to a particular CL? Part of the value of this script is to be able to re-run it periodically to see if it picks up any changes, and if you tie it to a particular CL, then that will never happen.

Reply:
Good point. Done.